scheds/rust/scx_layered/src/stats.rs

```diff
@@ -55,9 +57,9 @@ pub struct LayerStats {
     pub load: f64,
     #[stat(desc = "fraction of total load")]
     pub load_frac: f64,
-    #[stat(desc = "# tasks")]
+    #[stat(desc = "count of tasks")]
```

Comment:
Are these changes beacause doc doesn't allow hash sign?

Reply:
it's because the hash sign in the front indicates heading in markdown.